blueprint/index.md

```diff
@@ -0,0 +1,311 @@
+---
+title: Conversation model to SQL Blueprint
+author: ronan.watkins
+indextype: blueprint
+icon: blueprint
+image: images/erdiagram.png
+category: 6
+summary: |
+  This Genesys Blueprint provides an example of how to design an SQL database for storing JSON data, specifically from the [POST 
+  /api/v2/analytics/conversations/details/query](https://developer.genesys.cloud/api/rest/v2/analytics/#post-api-v2-analytics-conversations-details-query) endpoint. A project containing a dockerized postgres database and Node.js Typescript driver will be included as an example.
+---
+
+# Conversation model to SQL Blueprint  
+
+This Genesys Blueprint provides an example of how to design an SQL database for storing JSON data, specifically from the [POST 
+/api/v2/analytics/conversations/details/query](https://developer.genesys.cloud/api/rest/v2/analytics/#post-api-v2-analytics-conversations-details-query) endpoint. A project containing a dockerized postgres database and Node.js Typescript driver will be included as an example.
+
+![ERDiagram](images/erdiagram.png "ERDiagram")
+
+## Solution components
+* **Genesys Cloud** - A suite of Genesys cloud services for enterprise-grade communications, collaboration, and contact center management.
+* **PostgreSQL** - An open source object-relational database system.
+* **Docker** - A set of platform as a service products that use OS-level virtualization to deliver software in packages called containers.
+* **Node.js** - An open-source, cross-platform, back-end JavaScript runtime environment that runs on the V8 engine and executes JavaScript code outside a web browser.
```

Comment:
Should mention minimum or recommended versions for the items in this list.

Reply:
I would also link to their respective Websites.  Its just a courtesy in case someone is not familiar with the technology.